config/locales/en.yml

```diff
@@ -103,10 +112,32 @@ en:
               blank: Please enter your place of birth
             email:
               invalid: Please enter a valid email
+        steps/respondent/personal_details_form:
+          attributes:
+            full_name:
+              blank: Please enter the full name
+            previous_full_name:
+              blank: Please enter the previous name
+            dob:
+              blank: Please enter the date of birth
+            birthplace:
+              blank: Please enter the place of birth
+            address:
+              blank: Please enter the address
+            home_phone:
+              blank: Please enter the home phone
+            email:
+              invalid: Please enter a valid email
   helpers:
     fieldset:
       steps_applicant_personal_details_form:
         has_previous_name: Do you have a previous name?
+      steps_respondent_personal_details_form:
+        has_previous_name: Do you have a previous name?
```

Comment:
The applicant is the one filling in the form, so the respondent is a different person. I think this needs to be "Do they have a previous name?"

Reply:
Yep, good catch. Actually I've checked the prototype and in both, applicant and respondent, it just says "Previous name?" so I will change it to match the prototype.